plugins/infinispan91/pom.xml

```diff
@@ -156,6 +146,18 @@
 
```

Comment:
In this pom file there's still the clustered-counter dependency defining the version. Is this redundant?

Reply:
I'm not sure what you're asking about here. Should clustered counter not be included in the `infinispan91` plugin or are you asking about the `${project.version}`? Including the parent POM means that the POMs don't need to include the `${version.infinispan}` or `${version.narayanajta}` manually.

Reply:
The clustered-counters dependency still has `${infinispan.version}`. That's what I mean.

Reply:
OK, sorry. For some reason, the `infinispan-clustered-counter`, `infinispan-marshaller-protostuff`, and `infinispan-marshaller-kryo` dependencies are not included in the parent POM in 9.1.x. In 9.2.x, this works. I'll take a look at it again and add a comment to this file.